js/search.js

```diff
@@ -36,7 +49,33 @@ function(ytData, playlist, template, $) {
     }
 
     var displaySearchResults = function(){
-        $("#search_res").empty()
+        $("#search_res").empty();
+        if(!$.isEmptyObject(searchPlaylist)){
+            var tirow=template1.clone();
+            $(tirow).find('.ti').html('playlist');
+            tirow.appendTo('#search_res');
+            for(song in searchPlaylist){
+                var vidrow=template.getVidRow(searchPlaylist[song]);
+                $(vidrow).addClass("item-search");
+                $(vidrow).on("click",function(){
+                    //console.log('clicked '+this.id);
+                    var hashId="#playlist > #"+this.id;
+                    var target=$(hashId).offset().top-$("#playlist").offset().top;
```

Comment:
Here `$("#playlist").offset().top` is also not working too well, because the bounds of the `<ul>` don't give us the position of the whole list, because the list is much bigger than the `<ul>`. Thus using offset of `$("#playlist > li:nth-child(1)")` should work. ([nth-child ref](https://api.jquery.com/nth-child-selector/))